items/Project/Global/System.Workflows/Demo Workflow/Draft/Submit/Submission Notification.yml

```diff
@@ -38,30 +38,6 @@ Languages:
         Comments: $comment$ <br />
         Item path : $itemPath$ <br />
         Item language : $itemLanguage$ <br />
-        Item Version : $itemVersion$ <br /><br />
+        Item Version : $itemVersion$ <br />

```

Comment:
For this item, The dual `<br />` is important to add a blank line between the properties block and the next paragraph. Please revert this change.

Reply:
sure will revert the change